explainer.md

```diff
@@ -99,21 +89,20 @@ Here are some examples of app badging applied at the OS level:
 ![Android home screen badge](images/android-badge.png)
 <br>Android home screen badge
 
-Unlike document contexts, badges applied in "handle" contexts can be shown and
-updated even when there are no tabs or windows open for the app. So there are
-some special considerations for this use case, such as how to set the badge in
-response to a [Push message](https://www.w3.org/TR/push-api/), and how to scope
-a badge so that it appears on the desired set of pages or apps. The most
-commonly requested use case for "handle" badging is badging an app icon on the
-OS shelf, but it generalizes to non-installed sites as well.
+Unlike document badges, badges applied to apps can be shown and updated even
+when there are no tabs or windows open for the app. After much deliberation, we
```

Comment:
I'd prefer that we drop the  "After much deliberation"... we should just focus on explaining how it works, and explain we have two separate methods as part of their definition (i.e., because of "different scoping and lifetime considerations").